src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -384,6 +384,11 @@
     <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj">
       <UndefineProperties>%(ProjectReference.UndefineProperties);TargetGroup</UndefineProperties>
     </ProjectReference>
+    <ProjectReference Include="..\..\System.Security.Principal\src\System.Security.Principal.csproj">
+      <Aliases>System_Security_Principal</Aliases>
+    </ProjectReference>
+    <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj">
```

Comment:
Actually if I do not add that build starts to complain that IPrincipal argument is not CLS compliant. However on adding this reference the error goes away. I believe System.Threading.Thread also adds a csproj reference to System.Runtime